runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TransportationPointLocation.java

```diff
@@ -0,0 +1,199 @@
+	CART_TO_SHILO("Cart to - Shilo", new WorldPoint(2780, 3214, 0)),
+
+	//Canoes
+	CANOE_BARBVILLAGE("Canoe", new WorldPoint(3111, 3409, 0)),
+	CANOE_CHAMPIONSGUILD("Canoe", new WorldPoint(3202, 3344, 0)),
+	CANOE_EDGEVILLE("Canoe", new WorldPoint(3130, 3509, 0)),
+	CANOE_LUMBRIDGE("Canoe ", new WorldPoint(3241, 3238, 0)),
+
+	//Gnome Gliders
+	GNOME_GLIDER_KHARID("Gnome Glider", new WorldPoint(3278, 3213, 0)),
+	GNOME_GLIDER_APETOLL("Gnome Glider", new WorldPoint(2712, 2804, 0)),
+	GNOME_GLIDER_KARAMJA("Gnome Glider", new WorldPoint(2971, 2974, 0)),
+	GNOME_GLIDER_FELDIP("Gnome Glider", new WorldPoint(2540, 2969, 0)),
+	GNOME_GLIDER_GNOMESTRONGHOLD("Gnome Glider", new WorldPoint(2460, 3502, 0)),
+	GNOME_GLIDER_WHITEWOLF("Gnome Glider", new WorldPoint(2845, 3501, 0)),
+
+	//Balloons
+	BALLOON_VARROCK("Hot air Balloon", new WorldPoint(3298, 3480, 0)),
+	BALLOON_YANILLE("Hot air Balloon", new WorldPoint(2458, 3108, 0)),
+	BALLOON_GNOMESTRONGHOLD("Hot air Balloon", new WorldPoint(2478, 3459, 0)),
+	BALLOON_TAVERLEY("Hot air Balloon", new WorldPoint(2936, 3422, 0)),
+	BALLOON_FALADOR("Hot air Balloon", new WorldPoint(2921, 3301, 0)),
+
+	//Spirit Tree
+	SPIRITTREE_ARDOUGNE("Spirit Tree", new WorldPoint(2554, 3259, 0)),
+	SPIRITTREE_CORSAIR("Spirit Tree", new WorldPoint(2485, 2850, 0)),
+	SPIRITTREE_GNOMESTRONGHOLD("Spirit Tree", new WorldPoint(2459, 3446, 0)),
+	SPIRITTREE_GNOMEVILLAGE("Spirit Tree", new WorldPoint(2538, 3166, 0)),
+	SPIRITTREE_GRANDEXCHANGE("Spirit Tree", new WorldPoint(3184, 3510, 0)),
+
+	//Carpets
+	CARPET_KHARID("Carpet", new WorldPoint(3311, 3107, 0)),
+	CARPET_BEDABIN("Carpet", new WorldPoint(3183, 3042, 0)),
+	CARPET_POLLNIVNEACH_NORTH("Carpet", new WorldPoint(3351, 3001, 0)),
+	CARPET_POLLNIVNEACH_SOUTH("Carpet", new WorldPoint(3345, 2943, 0)),
+	CARPET_NARDAH("Carpet", new WorldPoint(3399, 2916, 0)),
+	CARPET_SOPHANEM("Carpet", new WorldPoint(3288, 2814, 0)),
+	CARPET_MENAPHOS("Carpet", new WorldPoint(3244, 2812, 0)),
+	CARPET_UZER("Carpet", new WorldPoint(3468, 3111, 0)),
+
+	//Teleports
+	TELEPORT_ARCHIVE_FROM_ARCEUUS("Teleport to - Library archive", new WorldPoint(1623, 3808, 0)),
+	TELEPORT_HARMELSS_FROM_HARMONY("Teleport to - Mos le Harmless", new WorldPoint(3784, 2828, 0)),
+	TELEPORT_RUNE_FROM_ARDOUGNE("Teleport to - Rune Eccense", new WorldPoint(2681, 3325, 0)),
+	TELEPORT_RUNE_FROM_YANILLE("Teleport to - Rune Eccense", new WorldPoint(2592, 3089, 0)),
+	TELEPORT_SORCERESS_GARDEN("Teleport to - Sorceress`s garden", new WorldPoint(3320, 3141, 0)),
+
+	//Other
+	ALTER_KOUREND_UNDERGROUND("Altar", new WorldPoint(1662, 10047, 0)),
+	FAIRY_RING_ZANRIS_TO_KHARID("Fairy Ring to - Al Kharid", new WorldPoint(2483, 4471, 0)),
+	FAIRY_RING_ZANRIS_TO_SHACK("Fairy Ring to - Shack", new WorldPoint(2451, 4471, 0)),
+	MOUNTAIN_GUIDE_QUIDAMORTEM("Mountain Guide", new WorldPoint(1275, 3559, 0)),
+	MOUNTAIN_GUIDE_WALL("Mountain Guide", new WorldPoint(1400, 3538, 0)),
+	MUSHTREE_MUSHROOM_FOREST("Mushtree", new WorldPoint(3674, 3871, 0)),
```

Comment:
The House on the Hill mushtree doesn't have an icon, but maybe it should?

Reply:
Do you want just a blank tooltip there than if there is no Icon present? Or do you want an icon placed for it?

Reply:
It would be better with the transportation icon